crypto/fipsmodule/rand/rand.c

```diff
@@ -395,6 +363,9 @@ void RAND_bytes_with_additional_data(uint8_t *out, size_t out_len,
     uint8_t seed[CTR_DRBG_ENTROPY_LEN];
     int used_cpu;
     rand_get_seed(state, seed, &used_cpu);
+
+    uint8_t add_data_for_reseed[CTR_DRBG_ENTROPY_LEN];
```

Comment:
This is only used in FIPS mode. So bit annoying to swallow extra memory usage for most of consumers.

But I can't think of a pretty way to avoid right now.